### ISSUE TITLE
Ranges for extraordinary abilities

### ISSUE THREAD

**Issue by Vorksector** [openlegend/core-rules#321]
I was reading the Combat chapter 6, and the max range of extraordinary attacks is 75 ft, and with an attribute score of 7-9. The thing is, for a mage, that range is quite short, I think the max range of extraordinary attacks should be modified to make more sense. Also, the disadvantage associated with area attacks: disadvantage 1 for a 5' cube, and for me that doesn't make any sense. If someone could at least explain the reason for the short range for extraordinary attacks, and why a 5' cube has disadvantage 1, I would greatly appreciate it

**Comment by ucffool** [comment]
Mightybell would be the place for discussions, FYI. 

> If someone could at least explain the reason for the short range for extraordinary attacks, and why a 5' cube has disadvantage 1, I would greatly appreciate it

Because you could single-target for no disadvantage, or area 5' cube for disadvantage 1... you have options, and sometimes that 5' cube means hitting more than 1 target.

**Comment by brianfeister** [comment]
If you are targeting a single target or a 5' cube, then you are not multi-targeting, so the text doesn't apply. 

75' for range is sufficient, you can use the " heightened invocation" feet to increase that range, and now that I think about it it's not a bad idea to add rules for extending extraordinary range via the feat that increases your range, I believe it's called Marksman.

**Comment by brianfeister** [comment]
Regarding disadvantage 1, that is a bug @istabosz. Should only mention 10' cube or bigger, 5' cube suffers no disadvantage.

**Comment by Vorksector** [comment]
With the marksman feat adding range to the extraordinary attacks, I think that would do the job. To close up, a 10' cube is disadvantage 1, a 15' cube is disadvantage 2, and so on? Or is it disadvantage 2 for 10' cube and so on?

**Comment by brianfeister** [comment]
10' cube (a multi target 2 attack) is disadvantage 2, 15' cube is Disadvantage 3. The problem with the rules text is that a 5' cube is not a multi target attack so it shouldn't be on the example. But beyond 1 target, disadvantage is equal to number of 5' interments in area length.

**Comment by brianfeister** [comment]
@istabosz was about to fix this myself and decided that it should have your wordsmith's finesse applied. I can see from a verbiage perspective why you wrote it this way, but I don't think 5' area attacks are multi-attacks, so we should have the verbiage reflect 10' / 2 targets +

**Comment by NicoFrancart** [comment]
I don't see the issue with targeting a 5' cube having disadvantage 1. Usually no one will use this option, but in exceptional circumstances why not? Imagine you're fighting a swarm of small creature, several enemies could occupy the same 5' cube and then this attack would be different from a single-target attack

**Comment by istabosz** [comment]
@brianfeister I was thinking the same thing as @NicoFrancart 

Is there a particular reason not to keep the disadvantage 1 for a 5' cube? It just seems neater to not have a special exception.

**Comment by apturtia** [comment]
The enemy swarm example could be added to area rules. I think it clarifies the intended rule quite nicely.

**Comment by brianfeister** [comment]
Ok, I can agree that it's tidier to not have the extra words to describe the exception to the rule, but I think an area
1 attack should have Disadvantage 0, so let's compromise by never saying what the penalty is for an area 1 attack and start at 2 for all explanations

**Comment by jcte02** [comment]
I may not be skilled with words, but I would suggest something on the line of `If you target an area wider than 5', your attack suffers disadvantage 1 per 5' increment`

**Comment by brianfeister** [comment]
I'm a big fan of that @jcte02

**Comment by apturtia** [comment]
`5' increment` gives me impression that area of 5'x5' cube is without disadvantage, 10'x'10 cube is only with 1 disadvantage (as in 1 increment). I would prefer current wording but add a small table to illustrate
5'x5' -> 1 (maybe leave this out)
10'x10' -> 2
15'x15' -> 3

**Comment by jcte02** [comment]
@apturtia Yes it is indeed ambiguous. What about `If you target an area wider than 5', your attack suffers disadvantage 1 per 5' of lenght`? It only adds a small phrase to the original wording.

**Comment by istabosz** [comment]
I'll figure out some wording that works. I just wanted to make sure that we wanted a 5' cube to be without disadvantage.

**Comment by brianfeister** [comment]
@istabosz nudge about this.

**Comment by istabosz** [comment]
Oh yea. I took care of this. I did it in the draft for the re-organized combat chapter which I hadn't implemented yet because of SRD concerns. I'll be sure to add all that in during the final edits. The revised text is as follows:

**Cube.** You target a cube with equal length, width, and height designated by you. The attack suffers disadvantage 1 per 5' length of the cube's sides. For example, a 10' cube suffers disadvantage 2, a 15’ cube suffers disadvantage 3, and so on. **Special Exception:** Targeting a 5’ cube suffers no disadvantage.
